### DISCUSSION TITLE
Replace the Tech3 org with this one that has the Pipi removed

### DISCUSSION THREAD

**Discussion by DrippingYellow** [DoukutsuRandamu/IssueTracker#93] [Suggestions]
My re-creation of the lost Tech3 sample org, which you included in the latest update as a new song for the factory in the second campaign, makes liberal use of "Pipi": An OrgMaker setting so obscure that Pixel didn't even bother to program it into Cave Story's implementation of the Organya engine. As a result, the countermelody (channels 3 and 4) sounds fine in OrgMaker, but is a little grating on the ears in-game.

I decided to make a variation of the Tech3 org with quadruple resolution and the Pipi removed, which sounds relatively the same as the OG org, so I was thinking you could replace the file used in-game with this: [Tech3_modified.org.zip](https://github.com/DoukutsuRandamu/IssueTracker/files/15478097/Tech3_modified.org.zip)

**Comment by TrashboxBobylev** [comment]
Satisfied in 1.44.1 (by adding pipi into Organya code, with help of Strultz).
